.travis.yml

```diff
@@ -30,3 +33,8 @@ script:
 
 after_failure:
     - for i in $(find tests -name \*.actual); do echo "--- $i"; cat $i; echo; echo; done
+
+cache:
+  directories:
+    - vendor
```

Comment:
Kazdou verzi Nette ale testuje na vice ruznych verzi PHP. Navic u ruznych verzi pouziva ``git checkout`` na lokalni kopii. Podle logu na Travisu mi prijde ze cachovani vendor slozky funguje.

Travis NETTE=master (PHP verze 5.4,  5.5,  5.6)
```
$ composer install --no-interaction --prefer-source
Loading composer repositories with package information
Installing dependencies (including require-dev)
Nothing to install or update
Writing lock file
Generating autoload files
```

Tedy slozka vendor se spravnymi balicky musela byt pritomna (ani to netaha ze svych composer cache). Kdyz se podivas na testy predchozich PR tak to vzdy clonujou.